Name: Shorcut-keys-In-Browser
Version: 1.3.5-1
Author: Fritos1406
Description: Adds shorcut keys to the launcher. Hold onto Gesture+key to execute
New Card: Gesture+n
Add Bookmark: Gesture+m
Add To Launcher: Gesture+l
Bookmarks: Gesture+b
I tried doing the history and the share shorcuts but they just ended up leaving the browser useless 

--- .orig/usr/palm/applications/com.palm.app.browser/app/controllers/page-assistant.js
+++ /usr/palm/applications/com.palm.app.browser/app/controllers/page-assistant.js
@@ -767,8 +767,8 @@
 		this.appMenuModel = {
 			visible: true,
 			items: [
-				MenuData.ApplicationMenu.NewCard,
-				MenuData.ApplicationMenu.AddBookmark,
+				{label:$L('New Card'),     shortcut:'n', command:'new-page-cmd'},
+				{label:$L('Add Bookmark'),     shortcut:'m', command:'add-bookmark-cmd'},
 				{
 					label: $L("Page"),
 					items: [
@@ -772,7 +772,7 @@
 				{
 					label: $L("Page"),
 					items: [
-						MenuData.ApplicationMenu.AddToLauncher,
+						{label:$L('Add To Launcher'),     shortcut:'l', command:'add-launch-icon-cmd'},
 						MenuData.ApplicationMenu.SharePage]
 				},
 				MenuData.ApplicationMenu.ShowBookmarks,
@@ -775,7 +775,7 @@
 						MenuData.ApplicationMenu.AddToLauncher,
 						MenuData.ApplicationMenu.SharePage]
 				},
-				MenuData.ApplicationMenu.ShowBookmarks,
+				{label:$L('Bookmarks'),     shortcut:'b', command:'show-bookmarks-cmd'},
 				MenuData.ApplicationMenu.ShowHistory]
 		};
 
